### ISSUE TITLE
Rajouter un status Invalide ou Hors Sujet

### ISSUE THREAD

**Issue by ogirardot** [LateralThoughts/lt-agora#18]
L'idée étant de refuser de statuer sur des propositions non SMART, non actionnables ou de vision pure...

**Comment by jblemee** [comment]
+1

En revanche que le lieu de débat soit sur Agora me semble bien. Cela permettra de créer des propositions smart à partir de débat non-smart.

**Comment by JunkieLand** [comment]
+1

**Comment by fbiville** [comment]
mais comment on arbitre ça ?
on crée un vote en parallèle ?

**Comment by ogirardot** [comment]
non, c juste un statut comme +1, -1 etc... si suffisamment de personnes considère que le sujet n'a pas à être traité là, il sera fermé.
On peut définir ensemble le "suffisamment"  :)
